exporter/opentelemetry-exporter-prometheus/src/opentelemetry/exporter/prometheus/__init__.py

```diff
@@ -112,8 +126,36 @@ class PrometheusMetricReader(MetricReader):
             the metric belongs to.
     """
 
-    def __init__(self, prefix: str = "") -> None:
-        super().__init__()
+    def __init__(
+        self,
+        preferred_temporality: Dict[type, AggregationTemporality] = None,
```

Comment:
Prometheus is strictly cumulative as mentioned below. Why do we need this option at all?

Reply:
From SIG call, DIego expressed concerns that the constructors are diverging. This is fine, and not violating the interface contracts e.g. https://stackoverflow.com/questions/5490824/should-constructors-comply-with-the-liskov-substitution-principle

Reply:
As agreed in the SIG, I have removed the parameters from the Prometheus constructor.